tst/regression/test_suites/particle_leapfrog_outflow/particle_leapfrog_outflow.py

```diff
@@ -49,10 +55,7 @@ def Analyse(self, parameters):
                 [-0.1, 0.3, 0.475, 0.0, 0.0, 0.5],
             ]
         )
-        final_data = structured_to_unstructured(
-            final_data[["x", "y", "z", "vx", "vy", "vz"]]
-        )
         if ref_data.shape != final_data.shape:
             print("TEST FAIL: Mismatch between actual and reference data shape.")
             return False
-        return (final_data == ref_data).all()
+        return (np.abs(final_data - ref_data) <= 1e-10).all()
```

Comment:
Maybe we need some kind of `test_utils.py` that defines things like `soft_equiv()`

Reply:
:+1: Yeah good idea. Not gonna add it here though.